flink-core/src/main/java/org/apache/flink/api/java/typeutils/PojoTypeInfo.java

```diff
@@ -264,6 +265,7 @@ public void getFlatFields(String fieldExpression, int offset, List<FlatFieldDesc
 	}
 
 	@Override
+	@PublicEvolving
```

Comment:
All the similar methods around it have the same stability. I think it should be fine to decrease it.